frontend/src/components/TransactionsList.tsx

```diff
@@ -28,79 +52,182 @@ function TransactionsList() {
+                        </p>
                       </div>
-                    )}
-                  </div>
-                  <div className="overflow-hidden mr-3">
-                    <div className="flex items-center gap-2 truncate dark:text-white">
-                      <p className="text-lg md:text-xl font-semibold">
-                        {type == "incoming" ? "Received" : "Sent"}
-                      </p>
-                      <p className="text-sm md:text-base truncate text-muted-foreground">
-                        {dayjs(tx.settled_at * 1000).fromNow()}
+                      <p className="text-sm md:text-base text-muted-foreground">
+                        {tx.description || "Lightning invoice"}
                       </p>
                     </div>
-                    <p className="text-sm md:text-base text-muted-foreground">
-                      {tx.description || "Lightning invoice"}
-                    </p>
-                  </div>
-                  <div className="flex ml-auto text-right space-x-3 shrink-0 dark:text-white">
-                    <div className="flex items-center gap-2 text-xl">
-                      <p
-                        className={`font-semibold ${
-                          type == "incoming" &&
-                          "text-green-600 dark:color-green-400"
-                        }`}
-                      >
-                        {type == "outgoing" ? "-" : "+"}{" "}
-                        {Math.floor(tx.amount / 1000)}
-                      </p>
-                      <p className="text-muted-foreground">sats</p>
+                    <div className="flex ml-auto text-right space-x-3 shrink-0 dark:text-white">
+                      <div className="flex items-center gap-2 text-xl">
+                        <p
+                          className={`font-semibold ${
+                            type == "incoming" &&
+                            "text-green-600 dark:color-green-400"
+                          }`}
+                        >
+                          {type == "outgoing" ? "-" : "+"}{" "}
+                          {Math.floor(tx.amount / 1000)}
+                        </p>
+                        <p className="text-muted-foreground">sats</p>
 
-                      {/* {!!tx.totalAmountFiat && (
+                        {/* {!!tx.totalAmountFiat && (
                         <p className="text-xs text-gray-400 dark:text-neutral-600">
                           ~{tx.totalAmountFiat}
                         </p>
                       )} */}
+                      </div>
                     </div>
                   </div>
-                </div>
-              </div>
+                </CredenzaTrigger>
+                <CredenzaContent className="slashed-zero">
+                  <CredenzaHeader>
+                    <CredenzaTitle>
+                      {type == "outgoing" ? "Sent Bitcoin" : "Received Bitcoin"}
```

Comment:
would it be possible to move this to a new component? this component is large already

Reply:
Done!